### PR TITLE
Added user manual for Xero, Quickbooks and Zohobooks modules

### PR DESCRIPTION
**Pull request by stojankukrika**
Added user manual for Xero, Quickbooks and Zohobooks modules

### CODE REVIEW

File: en/app-manual/import-export/quickbooks.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+![new field](_images/quickbooks-akaunting-connect.png)
+
+**Step4:** After connect to your Quickbooks account you can select from menu items which data you like to copy from your Quickbooks into Akaunting app.
+
+![new field](_images/quickbooks-sync-menu.png)
+
+## Need a custom functionality
+
+Need a custom functionality in [Akaunting](https://akaunting.com), or want to develop a new feature, feel free to email us on stojan.kukrika@gmail.com.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/quickbooks.md]: We can't accept this line as it's not related to the app.

File: en/app-manual/import-export/xero.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+**Step3:** Connect to your Xero account.
+
+**Step4:** After connect to your Xero account you can select from menu items which data you like to copy from your Xero into Akaunting app.
+
+![new field](_images/xero-sync-menu.png)
+
+## Need a custom functionality
+
+Need a custom functionality in [Akaunting](https://akaunting.com), or want to develop a new feature, feel free to email us on stojan.kukrika@gmail.com.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/xero.md]: We can't accept this line as it's not related to the app.

File: en/app-manual/import-export/zohobooks.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+**Step3:** Connect to your Zohobooks account.
+
+**Step4:** After connect to your Zohobooks account you can select from menu items which data you like to copy from your Zohobooks into Akaunting app.
+
+![new field](_images/zohobooks-sync-menu.png)
+
+## Need a custom functionality
+
+Need a custom functionality in [Akaunting](https://akaunting.com), or want to develop a new feature, feel free to email us on stojan.kukrika@gmail.com.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/zohobooks.md]: We can't accept this line as it's not related to the app.

File: en/MENU.md [hunk 4]
```diff
@@ -57,6 +57,9 @@
 	    * [Expenses](app-manual/hr/expenses)
 	* Import Export
 		* [OFX](app-manual/import-export/ofx)
+		* [Quickbooks](app-manual/import-export/quickbooks)
```
Comment (halimturkoglu) [inline en/MENU.md]: It should be `QuickBooks`.

File: en/MENU.md [hunk 5]
```diff
@@ -57,6 +57,9 @@
...
+		* [Zohobooks](app-manual/import-export/zohobooks)
```
Comment (halimturkoglu) [inline en/MENU.md]: It should be `Zoho Books`.

File: en/app-manual/import-export/quickbooks.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+**How to use?**
+
+**Step1:** Find [Here](https://akaunting.com/apps/quickbooks) "**Quickbooks**" on Akaunting app store and click _Install_. 
+
+**Step2:** Click Settings then "Quickbooks" in the right menu. 
+
+![new field](_images/quickbooks-start.png)
+
+**Step3:** Connect to your Qucikbooks account.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/quickbooks.md:18]: It should be `QuickBooks`

File: en/app-manual/import-export/quickbooks.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+**What's Quickbooks?** 
+
+QuickBooks is an accounting software package developed and marketed by Intuit. QuickBooks products are geared mainly 
+toward small and medium-sized businesses and offer on-premises accounting applications as well as cloud-based 
+versions that accept business payments, manage and pay bills, and payroll functions.
+
+**How to use?**
+
+**Step1:** Find [Here](https://akaunting.com/apps/quickbooks) "**Quickbooks**" on Akaunting app store and click _Install_.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/quickbooks.md]: It should be `QuickBooks`

File: en/app-manual/import-export/quickbooks.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+QuickBooks is an accounting software package developed and marketed by Intuit. QuickBooks products are geared mainly 
+toward small and medium-sized businesses and offer on-premises accounting applications as well as cloud-based 
+versions that accept business payments, manage and pay bills, and payroll functions.
+
+**How to use?**
+
+**Step1:** Find [Here](https://akaunting.com/apps/quickbooks) "**Quickbooks**" on Akaunting app store and click _Install_. 
+
+**Step2:** Click Settings then "Quickbooks" in the right menu.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/quickbooks.md]: It should be `QuickBooks`

File: en/app-manual/import-export/quickbooks.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+**Step2:** Click Settings then "Quickbooks" in the right menu. 
+
+![new field](_images/quickbooks-start.png)
+
+**Step3:** Connect to your Qucikbooks account.
+
+![new field](_images/quickbooks-akaunting-connect.png)
+
+**Step4:** After connect to your Quickbooks account you can select from menu items which data you like to copy from your Quickbooks into Akaunting app.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/quickbooks.md]: It should be QuickBooks

File: en/app-manual/import-export/quickbooks.md (hunk 7 above)
Comment (halimturkoglu) [inline en/app-manual/import-export/quickbooks.md]: It should be `Akaunting App Store`

File: en/app-manual/import-export/xero.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+Xero
+==========
+
+**What's Xero?** 
+
+Xero online accounting software for your business connects you to your bank, accountant, bookkeeper, and other business apps.
+
+**How to use?**
+
+**Step1:** Find [Here](https://akaunting.com/apps/xero) "**Xero**" on Akaunting app store and click _Install_.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/xero.md]: It should be `Akaunting App Store`

File: en/app-manual/import-export/zohobooks.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
+Zohobooks
+==========
+
+**What's Zohobooks?**
```
Comment (halimturkoglu) [inline en/app-manual/import-export/zohobooks.md]: It should be `ZohoBooks`

File: en/app-manual/import-export/zohobooks.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
+Zohobooks
+==========
+
+**What's Zohobooks?** 
+
+Zoho Books is online accounting software that manages your finances, automates business workflows, and helps you work collectively across departments.
+
+**How to use?**
+
+**Step1:** Find [Here](https://akaunting.com/apps/zohobooks) "**Zohobooks**" on Akaunting app store and click _Install_.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/zohobooks.md]: It should be `ZohoBooks` and `Akaunting App Store`

File: en/app-manual/import-export/zohobooks.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+
+**What's Zohobooks?** 
+
+Zoho Books is online accounting software that manages your finances, automates business workflows, and helps you work collectively across departments.
+
+**How to use?**
+
+**Step1:** Find [Here](https://akaunting.com/apps/zohobooks) "**Zohobooks**" on Akaunting app store and click _Install_. 
+
+**Step2:** Click Settings then "Zohobooks" in the right menu.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/zohobooks.md]: It should be `ZohoBooks`

File: en/app-manual/import-export/zohobooks.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+
+**How to use?**
+
+**Step1:** Find [Here](https://akaunting.com/apps/zohobooks) "**Zohobooks**" on Akaunting app store and click _Install_. 
+
+**Step2:** Click Settings then "Zohobooks" in the right menu. 
+
+![new field](_images/zohobooks-start.png)
+
+**Step3:** Connect to your Zohobooks account.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/zohobooks.md]: It should be ZohoBooks

File: en/app-manual/import-export/zohobooks.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+
+**Step4:** After connect to your Zohobooks account you can select from menu items which data you like to copy from your Zohobooks into Akaunting app.
```
Comment (halimturkoglu) [inline en/app-manual/import-export/zohobooks.md]: It should be `ZohoBooks`

**Review by halimturkoglu — CHANGES_REQUESTED** [review]

**Review by halimturkoglu — CHANGES_REQUESTED** [review]
There are a few more changes that need to be done.

**Comment by stojankukrika** [comment]
@halimturkoglu can you tell me what they are?

**Pull request merged**